integration_tests/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "commerce-sdk-integration-test",
+  "version": "1.4.1-beta.0",
+  "description": "Salesforce Commerce Cloud SDK Integration Test",
+  "homepage": "https://developer.commercecloud.com/s/",
+  "scripts": {
+    "redis": "docker rm --force redis-commerce-sdk; docker run -dp 6379:6379 --name redis-commerce-sdk redis:5-alpine",
+    "redis:shutdown": "docker rm --force redis-commerce-sdk",
+    "redis:debug": "docker rm --force redis-commerce-sdk-debug; docker run -dp 6379:6379 --name redis-commerce-sdk-debug redis:5-alpine && docker exec -it redis-commerce-sdk-debug redis-cli monitor",
+    "redis:debug:shutdown": "docker rm --force redis-commerce-sdk-debug",
+    "test:basic": "mocha \"tests/basic/*.test.js\" --retries 2 --timeout 5000",
+    "test:redis": "mocha \"tests/redis/*.test.js\" --retries 2 --timeout 5000",
+    "pack:sdk": "npm pack ../packages/core && npm pack ../packages/generator && npm install",
+    "test": "npm run test:basic && npm run test:redis",
+    "postinstall": "depcheck"
+  },
+  "mocha": {
+    "forbidOnly": true,
+    "colors": true
+  },
+  "devDependencies": {
+    "@commerce-apps/core": "file:commerce-apps-core-1.4.1-beta.0.tgz",
```

Comment:
How does this stay in sync?

Reply:
Tried npm_package_version var, did not work. Need to figure to a way to do this automatically. 

Currently If it is not in sync, build will fail..

Reply:
Solving this is really part of @dbarbato-sf story.  And kind of the bulk of that story I would say